src/Reflection/FunctionReflection.php

```diff
@@ -34,4 +34,6 @@ public function isBuiltin(): bool;
 
 	public function getAsserts(): Assertions;
 
+	public function getDocComment(): ?string;
```

Comment:
No, add a specific method for `@param-out`.

Reply:
Probably on ParameterReflectionWithPhpDocs.

Reply:
we have `getDocComment` on `ClassMemberReflection` and therefore I figured it would be good to also have it on `FunctionReflection`. having `getDocComment` on all relevant `*Reflection` classes is helpful to [build custom annotations which extension packages ](https://github.com/staabm/phpstan-dba/blob/7ce54d8d8938b08085bb63e760deb93b97e42104/src/PhpDoc/PhpDocUtil.php#L14-L39) sometimes need.

would you agree adding `getDocComment` on `FunctionReflection` as a separate PR meant for use in extensions?

I agree so for the `param-out` case a dedicated method would the way to go.

Reply:
Yes please, add it on FunctionReflection, and fix it in PhpFunctionFromParserNodeReflection + PhpMethodFromParserNodeReflection. In a separate PR.